ModsExportFormatTestTotalPages.xml

```diff
@@ -0,0 +1,22 @@
+<?xml version="1.0" encoding="UTF-8" standalone="yes"?>
+<mods:modsCollection xmlns:mods="http://www.loc.gov/mods/v3" xmlns:ns2="http://www.w3.org/1999/xlink" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:schemaLocation="http://www.loc.gov/standards/mods/v3/mods-3-6.xsd">
+    <mods:mods ID="test">
+        <mods:identifier type="citekey">test</mods:identifier>
+        <mods:genre>inbook</mods:genre>
```

Comment:
Why is there a new test file? This is not part of this pr, only importer/exporter test classes should be changed.